cmd/sentinel/cmd/sentinel.go

```diff
@@ -1475,44 +1475,58 @@ func (s *Sentinel) updateCluster(cd *cluster.ClusterData, pis cluster.ProxiesInf
 						log.Infow("added new standby db", "db", db.UID, "keeper", db.Spec.KeeperUID)
```

Comment:
Is it possible to remove the formatting for the parts that we did not modify? This is going to cause a lot of pain when we try to upstream, or rebase on the upstream repo.